pkgs/development/python-modules/natasha/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage, fetchPypi, python, pymorphy2, razdel, navec, slovnet
+, yagry, ipymarkup, pytest, isPy3k, lib }:
+buildPythonPackage rec {
+  pname = "natasha";
+  version = "1.6.0";
+  format = "setuptools";
+
+  disabled = !isPy3k;
+
```

Comment:
```suggestion
```
We no longer maintain python2, so this is redundant.

Reply:
Done!